Dependency updates 6-26-25

zacharybrown204 opened the pull request.

zacharybrown204 committed:
Dependency updates 6-26-25

zacharybrown204 committed:
Dependencies

Downgrading two missed caveat dependencies

zacharybrown204 committed:
Dependencies

Downgrading jest-environment-jsdom back because of failing unit tests.

zacharybrown204 committed:
Dependencies

Updating typescript target to fix some ts errors.

zacharybrown204 committed:
Dependencies

Reverted change for typescript target
Reverted @types-node version back to match our major version of node.
Typescript fixes in generateCategories.ts

zacharybrown204 committed:
Dependencies

Removing carrot from a couple of dependencies

zacharybrown204 committed:
Dependency

Updating terraform definitions

zacharybrown204 committed:
Dependencies

Ran into an issue where there were multiple certs for the UI in a single environment. Adding a flag to the data block to simply pick the most recent one.

zacharybrown204 committed:
Dependencies - leaving comments in the doc

zacharybrown204 commented:
Dependencies successfully built and deployed on exp1
https://app.circleci.com/pipelines/github/ustaxcourt/ef-cms/10879/workflows/686495ce-5bc4-464c-86bb-109475884445

zacharybrown204 committed:
Merge branch 'staging' into dependencies-6-26-25

jimlerza committed:
Merge branch 'staging' into dependencies-6-26-25

jimlerza commented on web-api/terraform/modules/batch/docker-image/package.json:
```diff
@@ -10,14 +10,14 @@
   "description": "",
   "devDependencies": {
-    "@types/node": "22.15.30"
+    "@types/node": "24.0.4"
```
Do we want to lock this along with our app's node version?

TomElliottFlexion replied:
Ditto

zacharybrown204 replied:
ahh good catch, yeah I think we should. I'll go revert that one

zacharybrown204 replied:
Okay I've reverted that change. Since it was just a dev dependency in the docker image I don't think we'll need to retest the deployment.

TomElliottFlexion commented on web-api/terraform/modules/batch/batch.tf:
```diff
@@ -103,7 +103,7 @@ EOF
 
 resource "aws_batch_compute_environment" "aws_batch_compute_environment" {
-  compute_environment_name = "compute_environment_${var.environment}_${var.current_color}_${var.region}"
+  name = "compute_environment_${var.environment}_${var.current_color}_${var.region}"
```
Good find

TomElliottFlexion reviewed (approved):
Nice job!

zacharybrown204 committed:
dependencies

Reverting dependency in docker-image package.json back to match our node version.
